File: exercises/03 annoying blobs/cs/AnnoyingBlobsTest/DataAdderTest.cs
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace AnnoyingBlobs
{
    [TestClass]
    public class DataAdderTest
    {
        [TestInitialize]
        public void SetUp()
        {
        }

        [TestCleanup]
        public void TearDown()
        {
        }

        [TestMethod]
        public void TestData()
        {
            var myDataAdder = new DataAdder();
            myDataAdder.DoSomeAdding();
        }
    }
}

File: solutions/03 annoying blobs/cpp/AnnoyingBlobs/AnnoyingBlobs tests/DataAdderTest.cpp
#include "gtest/gtest.h"
#include "gmock/gmock.h"
#include "DataAdder.h"

class DataAdderTest : public ::testing::Test
{
};

TEST_F(DataAdderTest, testData)
{
	DataAdder myDataAdder;
	myDataAdder.doSomeAdding();
}